types/sharedb/lib/sharedb.d.ts

```diff
@@ -46,11 +46,15 @@ export interface RawOp {
     src: string;
     seq: number;
     v: number;
-    op: Op[];
     m: any;
     c: string;
     d: string;
 }
+
+export type CreateOp = RawOp & { create: {type: string; data: any }; del: undefined; op: undefined; };
```

Comment:
Add a space before `type` - should have been caught by the linter, I'm unsure why it wasn't.
```suggestion
export type CreateOp = RawOp & { create: { type: string; data: any }; del: undefined; op: undefined; };
```